app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdEvent.kt

```diff
@@ -16,16 +16,16 @@ object HideKeyboard : ScanSimpleIdEvent() {
     get() = "Scan Simple Card:Hide keyboard"
 }
 
-object ShortCodeChanged : ScanSimpleIdEvent() {
+object EnteredCodeChanged : ScanSimpleIdEvent() {
   override val analyticsName: String
-    get() = "Scan Simple Card:Short code changed"
+    get() = "Scan Simple Card:Entered code changed"
```

Comment:
FYI, once we changed the analytics name we need to update the reports on Mixpanel to track the new analytics names as well.

Reply:
Will do this. Asked how to do this here : https://simpledotorg.slack.com/archives/CFK2PHJ1L/p1620124416023800